packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -796,6 +905,8 @@ abstract class Element implements BuildContext {
     _slot = newSlot;
     _depth = _parent != null ? _parent.depth + 1 : 1;
     _active = true;
+    if (parent != null) // Only assign ownership if the parent is non-null
+      _owner = parent.owner;
```

Comment:
Ah, I remember now, the constructor is called in a method whose signature would be disruptive to change.